docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,180 @@
+
+    The following information indicates that you have connected to the Nebula Graph services:
+
+    ```bash
+    Welcome to Nebula Graph (Version 5d10861)
+
+    (user@127.0.0.1) [(none)]>
+    ```
+
+## Check the Nebula Graph service status and ports
+
+Run the following command to list the service information of Nebula Graph, such as state and ports.
+
+```bash
+$ docker-compose ps
+```
+
+The following information is displayed:
+
+```bash
+Name                     Command                       State                                                   Ports
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+nebula-docker-compose_graphd_1      ./bin/nebula-graphd --flag ...   Up (health: starting)   0.0.0.0:32867->13000/tcp, 0.0.0.0:32866->13002/tcp, 3369/tcp, 0.0.0.0:3699->3699/tcp
+nebula-docker-compose_metad0_1      ./bin/nebula-metad --flagf ...   Up (health: starting)   0.0.0.0:32865->11000/tcp, 0.0.0.0:32864->11002/tcp, 45500/tcp, 45501/tcp
+nebula-docker-compose_metad1_1      ./bin/nebula-metad --flagf ...   Up (health: starting)   0.0.0.0:32863->11000/tcp, 0.0.0.0:32862->11002/tcp, 45500/tcp, 45501/tcp
+nebula-docker-compose_metad2_1      ./bin/nebula-metad --flagf ...   Up (health: starting)   0.0.0.0:32861->11000/tcp, 0.0.0.0:32860->11002/tcp, 45500/tcp, 45501/tcp
+nebula-docker-compose_storaged0_1   ./bin/nebula-storaged --fl ...   Up (health: starting)   0.0.0.0:32879->12000/tcp, 0.0.0.0:32877->12002/tcp, 44500/tcp, 44501/tcp
+nebula-docker-compose_storaged1_1   ./bin/nebula-storaged --fl ...   Up (health: starting)   0.0.0.0:32876->12000/tcp, 0.0.0.0:32872->12002/tcp, 44500/tcp, 44501/tcp
+nebula-docker-compose_storaged2_1   ./bin/nebula-storaged --fl ...   Up (health: starting)   0.0.0.0:32873->12000/tcp, 0.0.0.0:32870->12002/tcp, 44500/tcp, 44501/tcp
+```
+
+>**NOTE**: Nebula Graph provides services to the clients through port TCP3699 by default. You can adjust the port number by modifying the [network configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/3.configurations/3.meta-config/#networking_configurations).
+
+## Check the service data and logs
+
+All the data and logs of Nebula Graph are stored persistently in the `nebula-docker-compose/data` and `nebula-docker-compose/logs` directories.
+
+The structure of the directories is as follows:
+
+```text
+nebula-docker-compose/
+  |-- docker-compose.yaml
+  |-- data
+  |     |- meta0
+  |     |- meta1
+  |     |- meta2
+  |     |- storage0
+  |     |- storage1
+  |     `- storage2
+  `-- logs
+        |- meta0
+        |- meta1
+        |- meta2
+        |- storage0
+        |- storage1
+        |- storage2
+        `- graph
+```
+
+## Stop the Nebula Graph services
+
+You can run the following command to stop the Nebula Graph services:
+
+```bash
+$ docker-compose down -v
```

Comment:
I think `-v` will remove named volumes declared in the `volumes`.
it is not necessary.

Reply:
can you write in some data, dock compose down and up again to see the data are lost or not?

Reply:
I tested it and nothing went wrong. Wrote some data, shut down the servers, and turned them on again. The data is still there.

Reply:
I tested it and nothing went wrong. Wrote some data, shut down the servers, and turned them on again. The data is still there.